pkg/steps/multi_stage_test.go

```diff
@@ -118,7 +124,7 @@ func TestGeneratePods(t *testing.T) {
 		},
 		Namespace: "namespace",
 	}
-	step := newMultiStageTestStep(config.Tests[0], &config, nil, &jobSpec)
+	step := newMultiStageTestStep(config.Tests[0], &config, nil, "artifact_dir", &jobSpec)
```

Comment:
Where is this value used?

Reply:
Nowhere, but it triggers the conditional in your previous comment.